test-utils/src/main/java/com/revenuecat/purchases/utils/productStubs.kt

```diff
@@ -41,7 +41,8 @@ fun stubStoreProduct(
 @SuppressWarnings("EmptyFunctionBlock")
 fun stubPurchaseOption(
     id: String,
-    pricingPhases: List<PricingPhase> = listOf(stubPricingPhase())
+    duration: String,
```

Comment:
Hmm it's a bit strange to pass the duration here when this belongs to the `pricingPhase` model... Maybe we could have a different utility function called `stubPurchaseOptionWithOnePricingPhase` or something that accepts the pricing phase properties for convenience? NABD though, I think this is acceptable.